api/aperture/cloud/cmd/v1/cmd.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+// Messages for aperturectl → cloud controller communication.
+package aperture.cloud.cmd.v1;
```

Comment:
Let's drop the `cmd` part from path. (I regret using `cmd` name for regular controller apis, it's too generic)

Reply:
Made the changes